opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -254,6 +256,23 @@ namespace Opm {
     }
 
 
+    size_t EclipseGrid::activeIndex(size_t i, size_t j, size_t k) const {
+        return activeIndex( getGlobalIndex( i,j,k ));
+    }
+
+
+
+    size_t EclipseGrid::activeIndex(size_t globalIndex) const {
+        assertCellInfo();
```

Comment:
Can we change this with something like

    if (!hasCellInfo())
        return globalIndex;